### ISSUE TITLE
Can't update Genshin after Version Kiana (1.69.7 Stable)

### ISSUE THREAD

**Issue by KawaiiSoCKs** [neon-nyan/Collapse#100]
This occurs when the patch is nearly finished extracting (99-100%) I assumed it was a write or windows permissions issue, but everything seems in order as well as running the launcher as Administrator. Excluding the fact that trying to remove read-only from my Genshin folder doesn't work. (it just reverts after I've completed the process). I tried to add exceptions for the launcher in Avast or turn off anything that may be preventing me from removing read-only as well.

```
System.UnauthorizedAccessException: Access to the path 'E:\Genshin Impact' is denied.
   at System.IO.FileSystem.DeleteFile(String)
   at System.IO.File.Delete(String)
   at CollapseLauncher.InstallManager.Base.InstallManagerBase`1.<StartPackageInstallation>b__41_0() in E:\myGit\CollapseLauncher\CollapseLauncher\Classes\InstallManagement\BaseClass\InstallManagerBase.cs:line 264
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread, ExecutionContext, ContextCallback, Object)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread, ExecutionContext, ContextCallback, Object)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task&, Thread )
--- End of stack trace from previous location ---
   at CollapseLauncher.InstallManager.Base.InstallManagerBase`1.StartPackageInstallation() in E:\myGit\CollapseLauncher\CollapseLauncher\Classes\InstallManagement\BaseClass\InstallManagerBase.cs:line 223
   at CollapseLauncher.InstallManager.Genshin.GenshinInstall.StartPackageInstallation() in E:\myGit\CollapseLauncher\CollapseLauncher\Classes\InstallManagement\Genshin\GenshinInstall.cs:line 49
   at CollapseLauncher.Pages.HomePage.UpdateGameDialog(Object sender, RoutedEventArgs e) in E:\myGit\CollapseLauncher\CollapseLauncher\XAMLs\MainApp\Pages\HomePage.xaml.cs:line 936
```

**Comment by neon-nyan** [comment]
Hi @KawaiiSoCKs.

Thanks for reporting this to us. A patch is still underway to fix this issue.

**Comment by neon-nyan** [comment]
The fix has been shipped with 1.69.8 patch.
Please update your launcher and try again

**Comment by KawaiiSoCKs** [comment]
I can verify it was fixed on my end after this patch. Thank you so much!
